lib/customer-deposit-wallet/rest/Cardano/Wallet/Deposit/REST/Start.hs

```diff
@@ -119,3 +126,10 @@ newFakeBootEnv =
         Read.mockGenesisDataMainnet
         . mapBlock Read.EraValue
         <$> newNetworkEnvMock
+
+newBootEnv :: NetworkLayer IO (CardanoBlock StandardCrypto) -> WalletBootEnv IO
+newBootEnv nl =
+    WalletBootEnv
+        (show >$< stdoutTracer)
+        Read.mockGenesisDataMainnet
```

Comment:
- Do we use the incomplete `mockGenesisDataMainnet` for production as well? It shouldn't have any cons, so probably?
- For other networks, need to get filepath used [here](https://github.com/cardano-foundation/cardano-wallet/blob/0b8273d4c17c661f74a116e185559044bebe3d98/lib/exe/app/cardano-wallet.hs#L292-L297) with `Read.readGenesisData`. And, it would be nice to change https://github.com/cardano-foundation/cardano-wallet/blob/0b8273d4c17c661f74a116e185559044bebe3d98/lib/network-layer/src/Cardano/Wallet/Network/Config.hs#L66-L104 to be a pure function, but this would need to be done with care as it affects cardano-wallet